gui/wxpython/startup/guiutils.py

```diff
@@ -692,6 +696,58 @@ def delete_grassdb_interactively(guiparent, grassdb):
     return deleted
 
 
+def can_switch_mapset_interactively(guiparent, grassdb, location, mapset):
+    """
+    Checks if mapset is locked and offers to remove the lock file.
+
+    Returns True if user wants to switch to the selected mapset in spite of
+    removing lock. Returns False if a user wants to stay in the current
+    mapset or if an error was encountered.
+    """
+    can_switch = True
+
+    user = get_current_user()
+    lockfile = get_lockfile_if_present(grassdb, location, mapset)
+    timestamp = (datetime.datetime.fromtimestamp(
```

Comment:
You didn't test this properly, now you can't switch to mapset without a lockfile.

Reply:
That's strange. I am testing it now and it works fine.